api/database/migrations/2015_07_21_144551_create_articles_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateArticlesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create(\App\Models\Article::getTableName(), function (Blueprint $table) {
+                $table->uuid('article_id')->primary();
+                $table->string('title', 255);
+                $table->text('content');
+                $table->string('permalink', 255)->index()->nullable();
+                $table->dateTime('first_published')->nullable();
+
+                $table->dateTime('created_at');
+                $table->dateTime('updated_at')->nullable();
+            }
+        );
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::drop(\App\Models\Article::getTableName());
+    }
+}
```

Comment:
too hacky, tried that
1) need a place to store article.permalinks somewhere
2) need to add it back on some after save event
3) need to validate it somehow, though if it isn't listed in the attributes, then it's impossible right now

so if you want to leave fk, then let's just fire `ArticlePermalink->save()` it will give us 0 painful moments and 0 hacks (except  `ArticlePermalink->save()` itself )